spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -597,7 +588,7 @@ def get_decision_product(resp)
       it { expect(attention_with_barcode).to eq(false) }
     end
 
-    context 'with single barcode attention error' do
+    context 'with barcode attention error' do
```

Comment:
Do we still need to record this type of error, if `transaction_status` is taking it into account?

Reply:
in this case I opted to maintain existing behavior ... i think we could revisit this in future refactor of our error handling